packages/@coorpacademy-player-store/src/utils/state-extract.js

```diff
@@ -493,3 +493,11 @@ export const hasSeenLesson = (state: State, onPreviousSlide: boolean = false): b
     isEmpty(lessons) || any(ref => includes(ref, map('ref', lessons)), viewedResourcesForContent)
   );
 };
+
+export const getVideoUrl = (id: string) => (state: State): string =>
+  state && // eslint-disable-line lodash-fp/prefer-get
+  state.data &&
+  state.data.contents &&
+  state.data.contents.video &&
+  state.data.contents.video.entities &&
+  state.data.contents.video.entities[id];
```

Comment:
it seems that it's not in `state.data.contents` but in `state.data` directly:

https://github.com/CoorpAcademy/components/pull/1594/files#diff-df77b9ccae3ea57d6dc59267aae00b5fR48

Reply:
no, it's ok we'll use `fetchContent({type:video})`

Reply:
yes but the type is incorrect so (see the link above). the video state is not in contents state

Reply:
oh ok it was fixed after my first comment :) my bad